<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>493</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>587</width>
    <height>493</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>587</width>
    <height>493</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>311</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>311</width>
      <height>71</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; font-style:italic; color:#ffffff;&quot;&gt;Ayudantía 04&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>451</width>
      <height>111</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>451</width>
      <height>111</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; font-style:italic; color:#ffffff;&quot;&gt;Interfaces Gráficas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="secondButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>151</width>
      <height>71</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(224, 224, 224);
font: 22pt &quot;.SF NS Text&quot;;</string>
    </property>
    <property name="text">
     <string>Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="firstButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>71</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>151</width>
      <height>71</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(230, 230, 230);
font: 14pt &quot;.SF NS Text&quot;;</string>
    </property>
    <property name="text">
     <string>GameControl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>114</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>114</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 1, 7);
font: 18pt &quot;.SF NS Text&quot;;</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
